gridstatus/ercot_api/ercot_api.py

```diff
@@ -978,6 +990,62 @@ def get_spp_day_ahead_hourly(self, date, end=None, verbose=False):
 
         return data.sort_values(["Interval Start", "Location"]).reset_index(drop=True)
 
+    @support_date_range(frequency=None)
+    def get_dam_load_60_day_resources_as_offers(self, date, end=None, verbose=False):
```

Comment:
Should these two functions be combined into one so you can get both the load and gen data without having to download the same file twice?